packages/components/Atoms/Global/scss/settings/_button.scss

```diff
@@ -19,8 +19,11 @@ $rpl-button-danger-hover-background-color: lighten(rpl-color('danger'), 5%) !def
 $rpl-button-danger-text-color: rpl_color('white') !default;
 $rpl-button-active-text-color: $rpl-button-text-color !default;
 $rpl-button-active-background-color: transparent !default;
-$rpl-button-active-background-image: rpl_gradient('primary_gradient_90') !default;
+$rpl-button-primary-active-background-color: transparent !default;
+$rpl-button-primary-active-background-image: rpl_gradient('primary_gradient_90') !default;
```

Comment:
For backward compatibility, we may need to assign original value for this new one, so someone has set value for `$rpl-button-active-background-image` won't be impacted.
`$rpl-button-primary-active-background-image: $rpl-button-active-background-image !default;`.

BTW, do we need to add same stuff for secondary button?

Reply:
Ugh good find. I'll update this.
Secondary button will stay as is?
`$rpl-button-secondary-active-background-image: $rpl-button-active-background-image !default;`